.gitignore

```diff
@@ -5,22 +5,8 @@
 
 .idea
 
+deploy/pyodide
 deploy/vi
-
```

Comment:
Why did you remove these?

Reply:
This is standard gitignore file in viur base. The VI is served by the viur-cli.